PluginCore/PluginCore/FRService/FRSearch.cs

```diff
@@ -608,6 +609,45 @@ private List<SearchMatch> SearchSource(string src, int startIndex, int startLine
             return results;
         }
 
+        private bool LookupRegex(string ba, ref int i)
+        {
+            if (PluginBase.CurrentProject == null || 
```

Comment:
Of course I can make the change if you prefer, but it may be later today or already tomorrow, you caught me yesterday in the middle of another thing.